src/test/setupTests.ts

```diff
@@ -5,6 +5,18 @@ import "./matchers";
 global.window.scrollTo = jest.fn();
 jest.mock("react-ga4"); // Google Analytics requires a DOM.window which doesn't exist in test
 jest.mock("../app/services/websockets"); // MSW can't handle websockets just yet
+jest.mock("popper.js", () => {
```

Comment:
Commenting just to make sure that these changes are actually needed, as I cannot see the relationship with this PR just by looking at the code.

Reply:
This one was needed because the popper library wasn't playing well in tests. It seems to return some invalid styling and we get this in console:

```console.error
    Warning: `NaN` is an invalid value for the `left` css style property.```

It's specifically related to the `userEvent.hover` which triggers the tooltip popup.

Suggestions online suggested mocking it in jest. eg: https://github.com/react-bootstrap/react-bootstrap/issues/4997

Reply:
Thanks for explaining, Hannah! It looks like this block is now commented out 🤔

Reply:
Urgh 🙄 I commented it out to replicate the error again! Fixed now